rustls/src/error.rs

```diff
@@ -274,6 +274,7 @@ pub enum PeerIncompatible {
     Tls12NotOfferedOrEnabled,
     Tls13RequiredForQuic,
     UncompressedEcPointsRequired,
+    ExtendedMasterSecretExtensionRequired,
```

Comment:
Nit: please keep this in alphabetic order.

Reply:
Fixed